spec/index.adoc

```diff
@@ -1623,6 +1623,8 @@ The return result is an extended public key consisting of an ECDSA `public_key`,
 This call requires that the ECDSA feature is enabled, and the `canister_id` meets the requirement of a canister id.
 Otherwise it will be rejected.
 
+Invoking this API will cost cycles. The concrete cost is yet to be determined.
```

Comment:
Isn't that true for any of the management canister calls?

Reply:
@Dfinity-Bjoern ^^^

Reply:
Maybe there is a difference between "cycles deducted from canister" and "cycles must be send with the call". Not sure how to best put it...

Reply:
Ah, yes! If it behaves differently there it's worth pointing out. But why should it?

Reply:
We have some management canister calls which are free (as in, the canister doesn't need to send any cycles with the request, and any cycles they do send are refunded).